__tests__/app.test.js

```diff
@@ -50,17 +50,18 @@ describe("GET /api/topics", () => {
 describe("GET /api/articles/:article_id", () => {
   test("responds with an article object with the correct properties", () => {
     return request(app)
-      .get("/api/articles/2")
+      .get("/api/articles/3")
       .expect(200)
       .then(({ body }) => {
         expect(body).toMatchObject({
           title: expect.any(String),
           topic: expect.any(String),
           author: expect.any(String),
-          body: expect.any(String),
+          // body: expect.any(String),
           created_at: expect.any(String),
           votes: expect.any(Number),
           article_img_url: expect.any(String),
+          comment_count: "2"
```

Comment:
instead of updating an prior test, we should create a new test that drives the addition of the comment_count property to the articles